complete/src/main/java/com/example/restservice/dao/Queue.java

```diff
@@ -17,24 +26,21 @@
   @GenericGenerator(name = "uuid2", strategy = "uuid2")
   private String queueId;
 
+  private boolean isPasswordProtected;
+  private int queuePassword;
   @OneToMany(cascade = CascadeType.ALL, mappedBy = "queue")
   private List<User> users;
 
-  public List<User> getUsers() {
-    return users;
-  }
-
   protected Queue() {}
 
-  public String getQueueName() {
-    return queueName;
+  public Queue(String queueName)
+  {
+    this.queueName=queueName;
```

Comment:
proper whitespaces pls

Reply:
That's fine for now, I'm completely ignoring formatting. We would agree on some common formatting, and keep a check as a github action..